cmd/fetch.go

```diff
@@ -43,3 +47,21 @@ func Fetch(ctx *cli.Context) {
 
 	hw.Summarize()
 }
+
+func setSubmissionState(c *config.Config, problems []*api.Problem) error {
```

Comment:
That would probably be nice. Another thing we could do is have an alias type for `[]*api.Problem` that has a function to update the state. It could both make the call and do the loop. Not sure what that would look like in terms of testing, though.